ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/utils.py

```diff
@@ -46,6 +46,9 @@ def _underlay_links(self) -> list:
         """
         underlay_links = []
         underlay_links.extend(self._uplinks)
+        for uplink in underlay_links:
+            if default(get(uplink, "sflow"), self.shared_utils.fabric_sflow_uplinks) is not None:
+                uplink.update({"sflow": {"enable": default(get(uplink, "sflow"), self.shared_utils.fabric_sflow_uplinks)}})
```

Comment:
Since you no longer set anything in `eos_designs_facts` the `uplink` will never have an `sflow` key, so this can be simplified to:
```suggestion
        if self.shared_utils.fabric_sflow_uplinks is not None:
            for uplink in underlay_links:
                uplink.update({"sflow": {"enable": self.shared_utils.fabric_sflow_uplinks}})
```